tests/system/cluster/test_agent_key_polling/data/config.yml

```diff
@@ -0,0 +1,39 @@
+---
+wazuh-master:
+  - description: Enable remoted agent_key_polling at master side
+    after: <remote>
+    before: </remote>
+    content: |
+      <connection>secure</connection>
+      <port>1514</port>
+      <protocol>tcp</protocol>
+      <queue_size>131072</queue_size>
+      <key_polling enabled="yes">
+        <mode>local</mode>
+      </key_polling>
```

Comment:
Maybe this way of modifying the ossec configuration could done using the already implemented tools.